meta/main.yml

```
@@ -5,11 +5,12 @@ galaxy_info:
     Automatically initialise multiple network interfaces in a virtual machine
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  # points to the version of the ansible-core package
+  min_ansible_version: 2.13
```

```suggestion
  min_ansible_version: 2.10
```
This will match what we discussed